CHANGELOG.md

```diff
@@ -36,7 +36,7 @@ This can also be enabled programmatically with `warnings.simplefilter('default',
 * [`FPDF.write_html()`](https://py-pdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.write_html): fixed incoherent indentation of long `<ul>` list entries - _cf._ [issue #1073](https://github.com/py-pdf/fpdf2/issues/1073) - thanks to @lcgeneralprojects
 * default values for `top_margin` and `bottom_margin` in `HTML2FPDF._new_paragraph()` calls are now correctly converted into chosen document units.
 ### Removed
-* an obscure and undocumented [feature](https://github.com/py-pdf/fpdf2/issues/1198) of [`FPDF.write_html()`](https://py-pdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.write_html), which used to magically pass local variables as arguments.
+* an obscure and undocumented [feature](https://github.com/py-pdf/fpdf2/issues/1198) of [`FPDF.write_html()`](https://py-pdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.write_html), which used to magically pass class & instance properties as arguments.
```

Comment:
The previous description was better. If anything, then we could say "instance attributes" instead of "local variables".

 * [Properties](https://docs.python.org/3/library/functions.html#property) were not involved, at all.
 * The point of the [vars()](https://docs.python.org/3/library/functions.html#vars) builtin is specifically to return instance attributes
 * class attributes were in fact NOT passed on. This used to be claimed incorrectly in the documentation.